src/main/java/seedu/address/model/property/PropertyPricePredicate.java

```diff
@@ -28,11 +26,13 @@ public boolean test(Property property) {
         if (property.getAskingPrice() == null) {
             return false;
         }
-        NumberFormat format = NumberFormat.getCurrencyInstance();
+        // NumberFormat format = NumberFormat.getCurrencyInstance();
         int otherPrice;
         try {
-            otherPrice = format.parse(property.getAskingPrice().askingPrice).intValue();
-        } catch (ParseException e) {
+            otherPrice =
+                Integer
+                    .parseInt(property.getAskingPrice().askingPrice.replaceAll("[^\\d]+", ""));
```

Comment:
asking price might have decimals too, would using the regex defined in asking price help?

Reply:
I don't think any property prices bother including the cents part though, other than perhaps after tax calculations? But anyway will go add another field.

Reply:
changed to dollars + cents.